schemas/_common-schema.json

```diff
@@ -73,6 +73,61 @@
         }
       }
     },
+    "usr": {
+      "type": "object",
+      "description": "User properties",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Identifier of the user"
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the user"
+        },
+        "email": {
+          "type": "string",
+          "description": "Email of the user"
+        }
+      }
+    },
+    "connectivity": {
+      "type": "object",
+      "description": "Device connectivity properties",
+      "required": [
+        "status",
+        "interfaces"
+      ],
+      "properties": {
+        "status": {
+          "type": "string",
+          "description": "Status of the device connectivity",
+          "enum": ["connected", "not_connected", "maybe"]
```

Comment:
maybe sounds a bit strange, what cases does it cover?

Reply:
It comes from [iOS spec](https://developer.apple.com/documentation/network/nwpath/status/requiresconnection) where `requiresConnection` status is available:
> The path is not currently available, but establishing a new connection may activate the path.